src/game/lara/lara.c

```diff
@@ -513,6 +513,20 @@ void Lara_InitialiseInventory(int32_t level_num)
     RESUME_INFO *resume = &g_GameInfo.current[level_num];
 
     g_Lara.pistols.ammo = 1000;
+
+    if (g_GameInfo.remove_guns == true) {
```

Comment:
Let's drop the `== true` part, it's silly :D

Reply:
I always do this oof lol. Copy/paste things.